.github/workflows/package-and-upload-assets.yml

```
@@ -13,26 +11,51 @@ on:
 jobs:
-  package-and-upload:
+  build-and-upload:
     runs-on: ubuntu-latest
     steps:
-      # 1. Checks out your repository's code
       - name: Checkout code
         uses: actions/checkout@v4
 
-      # 2. Create TAR archive
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.9)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)